### PR TITLE
Translation - German and Italian Addition/Improvement

### PR DESCRIPTION
**Pull request by Nilia119**
**When merged this pull request will:**
- Add Missing German Translation
- Add Missing Italian Translation
_I had this lying around a couple of "days" and forgot to push..._

### CODE REVIEW

File: addons/compat_spe/stringtable.xml [hunk 1]
```diff
@@ -3,57 +3,93 @@
...
         </Key>
         <Key ID="STR_ACRE_compat_spe_M4A1_76_Command">
             <English>M4A1 76(W) Sherman (Command)</English>
+            <German>M4A1 76(W) Sherman (Kommando)</German>
+            <Italian>M4A1 76(W) Sherman (comando)</Italian>
         </Key>
         <Key ID="STR_ACRE_compat_spe_M4A0_75_Early_Command">
             <English>M4 Sherman (Command)</English>
+            <German>M4 Sherman (Kommando)</German>
+            <Italian>M4 Sherman (Commando)</Italian>
```
Comment (mrschick) [inline addons/compat_spe/stringtable.xml]:
```suggestion
            <Italian>M4 Sherman (Comando)</Italian>
```

File: addons/compat_spe/stringtable.xml [hunk 2]
```diff
@@ -3,57 +3,93 @@
...
         </Key>
         <Key ID="STR_ACRE_compat_spe_RadioSet1_Short">
             <English>A-Set</English>
+            <German>A-Kanäle</German>
+            <Italian>Canali A</Italian>
         </Key>
         <Key ID="STR_ACRE_compat_spe_RadioSet1_US">
             <English>Radio Set - Able</English>
+            <German>Kanäle - Able</German>
+            <Italian>Canali - In grado</Italian>
```
Comment (mrschick) [inline addons/compat_spe/stringtable.xml]: `Able` here is just a callsign, no point in translating it to a phrase, better to just leave it as-is like was done with the other Sets.
```suggestion
            <Italian>Canali - Able</Italian>
```

File: addons/sys_radio/stringtable.xml [hunk 3]
```diff
@@ -188,12 +200,18 @@
         </Key>
         <Key ID="STR_ACRE_sys_radio_ui_CurrentChannelBlock">
             <English>Current Channel Block</English>
+            <German>Aktueller Kanalblock</German>
+            <Italian>Blocco del canale corrente</Italian>
```
Comment (mrschick) [inline addons/sys_radio/stringtable.xml]: A `channel block` comprises multiple channels, so `channel` should have a plural meaning, no?
If so, I think this fits better.
```suggestion
            <Italian>Blocco di canali corrente</Italian>
```

File: addons/compat_spe/stringtable.xml [hunk 4]
```diff
@@ -3,57 +3,93 @@
...
         </Key>
         <Key ID="STR_ACRE_compat_spe_PzBefWgVI">
             <English>PzBefWg VI</English>
+            <German>PzBefWg VI</German>
+            <Italian>PzBefWg VI</Italian>
         </Key>
         <Key ID="STR_ACRE_compat_spe_RadioOperator">
             <English>Radio Operator</English>
+            <German>Funker</German>
+            <Italian>operatore radio</Italian>
```
Comment (mrschick) [inline addons/compat_spe/stringtable.xml]:
```suggestion
            <Italian>Operatore Radio</Italian>
```

File: addons/sys_radio/stringtable.xml [hunk 5]
```diff
@@ -161,21 +161,33 @@
         </Key>
         <Key ID="STR_ACRE_sys_radio_DefaultRadio_DisplayName">
             <English>Radio Item Replacement</English>
+            <German>Austausch des Funkgeräts</German>
+            <Italian>Sostituzione della radio</Italian>
         </Key>
         <Key ID="STR_ACRE_sys_radio_DefaultRadio_Description">
             <English>Default radio inventory items will be replaced with this type of radio.</English>
+            <German>Standard Funkinventargegenstände werden durch diesen Funkgerätetyp ersetzt.</German>
+            <Italian>Gli elementi dell'inventario radio predefiniti verranno sostituiti con questo tipo di radio.</Italian>
```
Comment (mrschick) [inline addons/sys_radio/stringtable.xml]: You could also use `radio predefinite`, but I believe this is more descriptive.
```suggestion
            <Italian>Le radio vanilla nell'inventario verranno sostituite con questo tipo di radio.</Italian>
```

**Review by mrschick — CHANGES_REQUESTED** [review]

File: addons/sys_radio/stringtable.xml (hunk 3 above)
Comment (Nilia119) [reply addons/sys_radio/stringtable.xml]: That was stupid...

File: addons/sys_radio/stringtable.xml (hunk 5 above)
Comment (Nilia119) [reply addons/sys_radio/stringtable.xml]: Yours sounds better!

**Review by mrschick — APPROVED** [review]

**Review by PabstMirror — APPROVED** [review]

**Comment by mrschick** [comment]
@Nilia119 I took a look at all other italian translations and revised them, some (especially `sys_gsa`) were pretty confusing.
I committed the changes to my fork at d01482204f33029eb1bef0b628bdc589418f2d31, if you think they are ok I'd appreciate it if they could be included in this PR.

**Comment by jonpas** [comment]
@mrschick can you open a new PR that includes changes from this PR as well as your additional changes on top?

**Comment by mrschick** [comment]
@jonpas Will do next week, when I've got the time 👍🏻

**Comment by mrschick** [comment]
Happened to find time for it now, just before bedtime.

**Pull request closed without merging**